test/distributed/test_c10d.py

```diff
@@ -1980,7 +1980,6 @@ def step_model(model, input, target):
 
         def update_parameters(model):
             for param in model.parameters():
-                param.data -= param.grad
```

Comment:
This one is actually modifying the value of `param` initially.
I think you want:
```
with torch.no_grad():
  param -= param.grad
```